app/views/groups/group_links/destroy.turbo_stream.erb

```diff
@@ -0,0 +1,27 @@
+<%= turbo_stream.append "flashes" do %>
+  <%= viral_flash(type:, data: message) %>
+<% end %>
+
+<%= turbo_stream.update "members", partial: "invited_groups" %>
+
+<%= turbo_stream.update "members_pagination" do %>
+  <%= render PaginationComponent.new(
```

Comment:
**suggestion:** This pagination is the same as that is in `create.turbo_stream.erb` and `index.turbo_stream.erb` why don't you just make a partial so you don't need to repeat all that?

Reply:
Since this is the case for the group_links and members I will do this in a separate pr

Reply:
Sounds good